native-android/src/state/hark.ts

```diff
@@ -0,0 +1,227 @@
+import _ from 'lodash';
```

Comment:
if we can we should trim this up to only what we need, also this is a bit outdated now since we use skeins.

Reply:
@patosullivan I wasn't sure if you had made React Native-specific changes to this file. Is it okay to copy `hark.ts` from the UI project?

Reply:
Yes, just make sure to copy this pattern for accessing the api:

```
  const { api } = useStore.getState();
    if (api === null) {
      return;
    }
 ```

Reply:
Actually, the new hark.ts uses react-query, I'm not sure if that's what we want in react native. I'll look into it.

Reply:
I think we should probably wait on the react-query stuff and just update this file to use skeins.

Reply:
The only thing we seem to be using it for currently is `sawRope` let's just rip everything out but that since we don't really need it.

Reply:
Will take care of this now!

Reply:
@arthyn `sawRope` calls `await get().update(rope.group)`, which pretty much requires the rest of the functions except for `sawSeam`. Is this update call safe to rip out as well?

Reply:
Sounds good! I've pared down the hark state file a bit and also fixed a `lodash` import in another file